src/debug/xdebugConnection.ts

```diff
@@ -206,7 +209,7 @@ export class ClassLineBreakpoint extends LineBreakpoint {
       this.line = parseInt(breakpointNode.getAttribute("lineno"), 10);
       this.fileUri = breakpointNode.getAttribute("filename");
     } else {
-      super(rest[0], rest[1]);
+      super(rest[0], rest[1], rest[4]);
```

Comment:
why not just
`parseInt(rest[4], 10)`
parseInt will return `NaN` for anything else